lib/expr_executor.cpp

```diff
@@ -252,16 +252,17 @@ namespace {
     }
   }
 
-  /* To avoid the following warning, which occur in uint32_t and uint64_t.
+  /* To avoid the following warning, which occur in uintN_t.
    *
    * warning C4146: unary minus operator applied to unsigned type, result still
    * unsigned
```

Comment:
Could you update why this is needed?

The current description is the original motivation but it's not different now. We do this for `X / -1 == -X`.